sdk/contentsafety/Azure.AI.ContentSafety/src/Customizations/ContentSafetyClientBuilderExtensions.cs

```diff
@@ -1,13 +1,62 @@
 // Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License.
 
+using System;
+using Azure;
+using Azure.AI.ContentSafety;
 using Azure.Core;
+using Azure.Core.Extensions;
 
 namespace Microsoft.Extensions.Azure
 {
     /// <summary> ContentSafety client builder extensions. </summary>
     [CodeGenModel("AIContentSafetyClientBuilderExtensions")]
     public partial class ContentSafetyClientBuilderExtensions
```

Comment:
Same here. It seems that before this file was only used for renaming the generated model.